src/plugins/data_source_management/public/components/constants.tsx

```diff
@@ -17,3 +18,29 @@ export const NO_DATASOURCES_CONNECTED_MESSAGE = 'No data sources connected yet.'
 export const CONNECT_DATASOURCES_MESSAGE = 'Connect your data sources to get started.';
 export const NO_COMPATIBLE_DATASOURCES_MESSAGE = 'No compatible data sources are available.';
 export const ADD_COMPATIBLE_DATASOURCES_MESSAGE = 'Add a compatible data source.';
+
+export const OPENSEARCH_DOCUMENTATION_URL =
+  'https://opensearch.org/docs/latest/dashboards/management/data-sources/';
```

Comment:
don't do this. use doc link service: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/core/public/doc_links/doc_links_service.ts

it's versioned